ci/run_testsuite_and_record_V2.sh

```diff
@@ -59,7 +59,7 @@ else
 fi
 
 # give mreg some time to create the database schema and start up
-sleep 5s
+sleep 5
```

Comment:
Uhm, this snuck in, but we actually do need to remove the "s" for the script to work on MacOS.

Reply:
I think `s` implied most places, so this shouldn't hurt anything. 

Which are famous last words, but oh well.